app/main/views/marketplace.py

```diff
@@ -245,12 +245,7 @@ def get_brief_by_id(framework_slug, brief_id):
 def get_brief_response_preview_by_id(framework_slug, brief_id):
     briefs = data_api_client.get_brief(brief_id)
     brief = briefs.get('briefs')
-
-    application_url = url_for('main.index', _external=True) + '2/brief/{}'.format(brief['id'])
-    if brief['lotSlug'] == 'digital-outcome':
```

Comment:
how about training?

Reply:
A static `.docx` template is used for training briefs.  At this point no dynamic data is generated for them.